Resources/Prototypes/Entities/Objects/Specific/Janitorial/soap.yml

```diff
@@ -180,6 +180,7 @@
         - ItemMask
   - type: DeleteOnTrigger
   - type: EmitSoundOnTrigger
+    predicted: true
```

Comment:
So this is an existing bug, so maybe just TODO this (because it's kinda a Balance Change™), but this needs a TriggerOnStep to actually work. It'll also need its audio to be set as positional if the DeleteOnTrigger stays. Additionally, this volume makes the splat effect literally inaudible for me. Maybe -8 or something would be better.

(The volume issue might actually be due to an engine issue I have yet to properly document, but it seems like different OpenAL implementations might not have consistent implementations of AL_GAIN which is... absurd. I'd be curious to know if the sound is audible for you as-is.)